src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.constants.tsx

```diff
@@ -0,0 +1,30 @@
+export const defaultOffer = {
```

Comment:
we already have `defaultResponse` const. can reuse it and not to create a new variable for it